File: 251111/A 부터 B 까지/a-to-b.js
const fs = require('fs');
const input = fs.readFileSync(0).toString().trim().split(' ');
const [A, B] = input.map(Number);

let result = '';
let i = A;
while (i <= B) {
    result += `${i} `;
    if (i % 2 !== 0) {
        i *= 2;
    } else {
        i += 3;
    }
}
console.log(result);

File: 260312/A 부터 B 까지/a-to-b.py
import sys

input = sys.stdin.readline

# 선언부

# 입력부
A, B = map(int, input().split())

# 호출부
while A <= B:
    print(A, end=' ')
    if A % 2 == 1:
        A *= 2
    else:
        A += 3